test/runtests.jl

```diff
@@ -1263,6 +1263,84 @@ end
             end
         end
 
+        # Here we check that the types of p_prototype and d_prototype actually 
```

Comment:
I guess we should also keep this test (without the d_prototype parts)

Reply:
Yes 👍